servers/visual/shader_language.cpp

```diff
@@ -571,7 +571,7 @@ ShaderLanguage::Token ShaderLanguage::_get_token() {
 					if (hexa_found) {
 						//hex integers eg."0xFF" or "0x12AB", etc - NOT supported yet
 						return _make_token(TK_ERROR, "Invalid (hexadecimal) numeric constant - Not supported");
-					} else if (period_found || float_suffix_found) {
+					} else if (exponent_found || period_found || float_suffix_found) {
```

Comment:
Just a matter of style, but maybe move it to the second place to match line 617?

Reply:
yeah makes sense, I'll change that once we make a decision on the other issue.